WcaOnRails/app/assets/javascripts/registrations.js

```diff
@@ -28,6 +28,11 @@ onPage('registrations#edit_registrations', function() {
     if(!confirm("Delete the " + $selectedRows.length + " selected registrations?")) {
       e.preventDefault();
     }
+
+    var $reason = $('#delete-reason').val()
+    var $comment = $('#delete-comment').val()
+    console.log($reason);
```

Comment:
Just noting that these two `console.log` should be removed before the feature goes live